bin/clean-up-docker-containers.sh

```diff
@@ -1,2 +1,6 @@
 #!/usr/bin/env bash
-docker rmi khalisfoundation/banidb-dev
+
+docker kill banidb-api
```

Comment:
what container is this?

Reply:
your container. image is banidb-dev but you named the run banidb-api. I just copied the kill/rm from the stop script so that clean will really clean.